README.md

```diff
@@ -1,3 +1,17 @@
+# DEPRECATED!
+
+hexo-renderer-swig has been deprecated.
+
+The [swig](https://node-swig.github.io/swig-templates/) project seems inactive. We recommend migrate to other template engines. (e.g. [nunjucks](https://mozilla.github.io/nunjucks/))
+
+Below references may help migrate from swig to nunjucks.
+
+> [JS Templating: Transitioning from swig to nunjucks](https://medium.com/engineers-optimizely/js-templating-transitioning-from-swig-to-nunjucks-ac0e94d1794b)
+> [hexo-renderer-nunjucks](https://github.com/hexojs/hexo-renderer-nunjucks)
+> [hexo-theme-next: Migrate to Nunjucks #1214](https://github.com/theme-next/hexo-theme-next/issues/1214)
+> [hexo-theme-next: Use nunjucks-compatible syntax & operator #1068](https://github.com/theme-next/hexo-theme-next/pull/1068/files)
+> [hexo-theme-next: Migrate to Nunjucks #1215](https://github.com/theme-next/hexo-theme-next/pull/1215/files)
```

Comment:
@stevenjoezhang @tomap 
I add references for migration to nunjucks. Is it enough?